src/server/config.rs

```diff
@@ -26,6 +26,7 @@ pub use storage::Config as StorageConfig;
 pub const DEFAULT_CLUSTER_ID: u64 = 0;
 pub const DEFAULT_LISTENING_ADDR: &str = "127.0.0.1:20160";
 const DEFAULT_ADVERTISE_LISTENING_ADDR: &str = "";
+const DEFAULT_STATUS_ADDR: &str = "127.0.0.1:9520";
```

Comment:
em, why 9520 as the default port, any reason? IMO, I prefer 20180 like 20160.

Reply:
Oh, I found here is a list related to Prometheus port. At that time `9520` was free to use. https://github.com/prometheus/prometheus/wiki/Default-port-allocations